feat: rework trigger workflow docs (#764)
knocklabs/docs: content/send-notifications/triggering-workflows/audiences.mdx

tags: ["trigger", "audience"]
section: Send notifications
---

<Callout
  emoji="🚧"
  text={
    <>
      Audiences is currently in beta. If you'd like early access, or this is
      blocking your adoption of Knock, please{" "}
      <a href="mailto:support@knock.app?subject=Audiences beta access">
        get in touch
      </a>
      .
    </>
  }
/>

Audience workflow triggers allow you to execute a workflow run when a user is added to a specific audience.

An audience is a collection of users who share a common characteristic. For example, you might have an audience of users who have signed up for a paid plan, or an audience of users who have made a purchase in the past 30 days.

To use audience triggers, you'll need to have an [audience](/concepts/audiences) created in Knock. You can create an audience by going to the "Audiences" section in the dashboard and clicking the "Create audience" button.

## Configuring an audience trigger

You can configure an audience trigger by selecting the "Audience" trigger type in the workflow builder and then selecting the audience you'd like to trigger the workflow on.

> Remember: you must have an audience created in Knock before you can use it as a trigger source.

Reviewer: ```suggestion
Remember: you must have an audience created in Knock before you can use it to trigger workflows.
```

## Frequently asked questions

<AccordionGroup>
  <Accordion title="Can I trigger a workflow to all users in an audience?">

>     No, you can currently only trigger a workflow to any **new users** who have
>     been added to an audience.

Reviewer 2: ```suggestion
    No, you can currently only trigger a workflow to any **new audience members** as they
    are added to an audience.
```
I think we can be a bit clearer about the semantics here

  </Accordion>
  <Accordion title="Can I specify multiple audiences as a trigger source of a workflow?">
    No, you can currently only specify a single audience as a trigger source for
    a workflow.
  </Accordion>
  <Accordion title="Can I trigger a workflow for a user who is removed from an audience?">
    No, you can currently only trigger a workflow for users who are added to an
    audience.
  </Accordion>
  <Accordion title="Can I prevent a workflow from being run if the user has already run through the workflow?">

>     Yes, you can use the workflow trigger frequency setting to control if a

Reviewer 2: ```suggestion
    Yes, you can use the [workflow trigger frequency setting](/send-notifications/triggering-workflows/overview#controlling-workflow-trigger-frequency) to control if a
```
